app/uk/gov/hmrc/tai/controllers/taxCodeChange/TaxCodeChangeController.scala

```diff
@@ -63,4 +64,19 @@ class TaxCodeChangeController @Inject()(authentication: AuthenticationPredicate,
         case ex: BadRequestException => BadRequest(Json.toJson(Map("reason" → ex.getMessage)))
       }
   }
+
+  def mostRecentTaxCodeRecords(nino: Nino, year: TaxYear): Action[AnyContent] = authentication.async {
```

Comment:
Since the response is JSON, should we be specifying that here? I've seen this similar pattern in https://github.com/hmrc/tai/blob/master/app/uk/gov/hmrc/tai/controllers/JourneyCacheController.scala#L50